nemo/core/classes/modelPT.py

```diff
@@ -239,6 +230,72 @@ def setup_test_data(self, test_data_config: Union[DictConfig, Dict]):
         """
         raise NotImplementedError()
 
+    def setup_multiple_validation_data(self, val_data_config: Union[DictConfig, Dict]):
+        """
+        (Optionally) Setups data loader to be used in validation, with support for multiple data loaders.
+
+        Args:
+            val_data_layer_config: validation data layer parameters.
+        """
+        try:
+            # Set some placeholder overriden by helper method
+            self._validation_loss_idx = 0
+            self._validation_names = None
+            self._validation_dl = None  # type: torch.utils.data.DataLoader
+
+            if val_data_config is not None:
+                if isinstance(val_data_config, dict):
+                    val_data_config = DictConfig(val_data_config)
+
+                self._cfg.validation_ds = val_data_config
+
+            model_utils.resolve_validation_dataloaders(model=self)
+
+            if self._validation_names is None:
+                if self._validation_dl is not None and type(self._validation_dl) in [list, tuple]:
+                    self._validation_names = ['val_{}_'.format(idx) for idx in range(len(self._validation_dl))]
+
+        except Exception:
+            logging.warning(
+                "Unable to load the validation data loader(s) with the provided config \n"
+                f"{self._cfg.validation_ds}\n"
+                f"Please call the ModelPT.setup_multiple_validation_data() method "
+                f"and provide a valid configuration file."
+            )
+
+    def setup_multiple_test_data(self, test_data_config: Union[DictConfig, Dict]):
+        """
+        (Optionally) Setups data loader to be used in test, with support for multiple data loaders.
+
+        Args:
+            test_data_layer_config: test data layer parameters.
+        """
+        try:
+            # Set some placeholder overriden by helper method
+            self._test_loss_idx = 0
+            self._test_names = None
+            self._test_dl = None  # type: torch.utils.data.DataLoader
+
+            if test_data_config is not None:
+                if isinstance(test_data_config, dict):
+                    test_data_config = DictConfig(test_data_config)
+
+                self._cfg.test_ds = test_data_config
+
+            model_utils.resolve_test_dataloaders(model=self)
+
+            if self._test_names is None:
+                if self._test_dl is not None and type(self._test_dl) in [list, tuple]:
+                    self._test_names = ['test_{}_'.format(idx) for idx in range(len(self._test_dl))]
+
+        except Exception:
```

Comment:
Please print the Exception here as well.

Reply:
Done